atom/app/node_main.cc

```diff
@@ -2,8 +2,6 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
-#ifdef ENABLE_RUN_AS_NODE
```

Comment:
Should this file be excluded from build if we build without `ENABLE_RUN_AS_NODE `?

Reply:
The files are already behind `enable_run_as_node` , hence I removed this guard.